### ISSUE TITLE
Map arnagoDb Attributes to C# properties

### ISSUE THREAD

**Issue by namnik** [yojimbo87/ArangoDB-NET#34]
My entity:
```
public class User
{
        [AliasField("_key")]
        public string Id { get; set; }

        [AliasField("name")]
        public string Name { get; set; }
}
```
`AliasField` works fine when **insert** a entity object:
```
var newUser = new User() { Id= "@9302794244" , Name = "Saman"};
db.Document.Create("User", newUser );
```
but when I **fetch** Data from User Collection, just fetch data of **Name field** and Id value is null:
```
var items=  db.Query.Aql("for item in User return item").ToList<User>().Value;
```
========>result:
{
**Id = null** ,
 Name = "Saman"
}
*** this problem also exists in two fields of edge collection**(_from , _to)**.  and return null value!
....؟!

**Comment by yojimbo87** [comment]
Thanks for the report, will fix it during the week.

**Comment by namnik** [comment]
thanks for responsibility. I will waiting ...

**Comment by yojimbo87** [comment]
Fixed in v0.10.1
